debian/syncd.init

```diff
@@ -20,16 +24,31 @@ start)
         [ -e /dev/linux-bcm-knet ] || mknod /dev/linux-bcm-knet c 122 0
```

Comment:
wonder if we have a function define like 

start_bcm () 
{
   [ -e /dev/linux-bcm-knet ] || mknod /dev/linux-bcm-knet c 122 0
          [ -e /dev/linux-user-bde ] || mknod /dev/linux-user-bde c 126 0		          [ -e /dev/linux-user-bde ] || mknod /dev/linux-user-bde c 126 0
          [ -e /dev/linux-kernel-bde ] || mknod /dev/linux-kernel-bde c 127 0		          [ -e /dev/linux-kernel-bde ] || mknod /dev/linux-kernel-bde c 127 0
   		   
          DAEMON_ARGS="-p /etc/syncd.d/dell_s6100.profile -N"		          DAEMON_ARGS="-p /etc/syncd.d/dell_s6100.profile -N"
 
         if [ -x $DAEMON_DSSERVE ]; then
             DAEMON=$DAEMON_DSSERVE
             DAEMON_ARGS="$DAEMON_DSSERVE_ARGS $DAEMON_ARGS"
         else
             DAEMON=$DAEMON_SYNCD
         fi
}